tools/autocorres/README.md

```diff
@@ -16,7 +16,7 @@ abstracts the result to produce a result that is (hopefully)
 more pleasant to reason about.
 
   [1]: https://www.cl.cam.ac.uk/research/hvg/Isabelle/
-  [2]: https://ts.data61.csiro.au/software/TS/c-parser/
+  [2]: https://ts.data61.csiro.au/software/TS/c-parser.html
```

Comment:
This one's wrong: the trailing slash is required.

Reply:
Looks like there's an error in the web copying.  I'll fix it in the website.

Reply:
I've fixed this one upstream now.

Reply:
Thanks! I'll switch it back to `https://ts.data61.csiro.au/software/TS/c-parser/`

Reply:
Cool.
With that should be OK to merge ...